src/pages/Team/components/List/style.css

```diff
@@ -147,3 +152,22 @@ p {
   }
 }
 
+.divPag {
```

Comment:
this classname is not enough clear for me

Reply:
removed container div with classname divPag.